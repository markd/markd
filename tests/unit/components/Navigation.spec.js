import { shallowMount, createLocalVue } from '@vue/test-utils'
import Component from '@/components/Navigation.vue'

import Vuex from 'vuex'
import VueRouter from 'vue-router'

const localVue = createLocalVue()
localVue.use(VueRouter)
localVue.use(Vuex)

const router = new VueRouter()

const store = new Vuex.Store()

describe('Navigation.vue', () => {
  it('renders', () => {
    const wrapper = shallowMount(Component, {
      localVue,
      store,
      router
    })

    expect(wrapper).toMatchSnapshot()
  })
})
