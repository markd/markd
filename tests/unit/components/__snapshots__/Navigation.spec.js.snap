// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation.vue renders 1`] = `
<nav class="navigation">
  <div class="navigation__searchform"><input type="text" placeholder="Search" class="navigation__searchformInput"></div>
  <div class="filterOptions"><button class="filterOptions__option">
      List
    </button></div>
</nav>
`;
